frontends/web/src/routes/account/send/send.tsx

```diff
@@ -314,7 +314,13 @@ class Send extends Component<Props, State> {
         });
     }
 
-    private txProposal = (updateFiat, result) => {
+    private txProposal = (updateFiat: boolean, result: {
```

Comment:
why is the result in .

```
apiPost('account/' + this.getAccount()!.code + '/tx-proposal', txInput)
            .then(result => {
```

not required to be typed? it should be the same type as this result here.

Reply:
Because `request.js` and `qttransport.js` have not been converted to TypeScript yet and return `any`, see comment in commit and PR.

`/tx-proposal` should be typed and moved to `api/account`. But I guess in a different PR.